erpnext/stock/doctype/landed_cost_voucher/test_landed_cost_voucher.py

```diff
@@ -133,7 +133,24 @@ def test_landed_cost_voucher_for_serialized_item(self):
 
 		set_perpetual_inventory(0)
 
-def submit_landed_cost_voucher(receipt_document_type, receipt_document):
+	def test_landed_cost_voucher_for_odd_numbers (self):
+		set_perpetual_inventory(1)
+
+		pr = frappe.copy_doc(pr_test_records[0])
+		pr.append("items", {
+			"item_code": "_Test Item",
+			"warehouse": "_Test Warehouse - _TC",
+			"cost_center": "_Test Company - _TC",
+			"qty": 5,
+			"rate": 50
+		})
+		pr.submit()
+
+		submit_landed_cost_voucher("Purchase Receipt", pr.name, 123.22)
+
```

Comment:
Add a `assertEqual` statement as well, to compare the distributed charges value.